pkg/gds/store/mockdb/mockdb.go

```diff
@@ -1,125 +1,159 @@
 package mockdb
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/trisacrypto/directory/pkg/gds/models/v1"
 	"github.com/trisacrypto/directory/pkg/gds/store"
 	"github.com/trisacrypto/directory/pkg/gds/store/iterator"
 	pb "github.com/trisacrypto/trisa/pkg/trisa/gds/models/v1beta1"
 )
 
-var _ store.Store = &MockDB{}
+var db store.Store = &MockDB{}
```

Comment:
The line of code `var _ store.Store = &MockDB{}` is a compiler based interface check; e.g. this noop will fail at compile time if `MockDB` doesn't implement `store.Store`. 

Keeping a singleton `db` and `state` around is a pattern in our other packages, so I'm fine with the change I just wanted to mention why the previous line was there before!

Reply:
Makes sense, thanks for the clarification.